src/test/client.rs

```diff
@@ -497,35 +497,6 @@ async fn saslprep_options() {
     auth_test_options("IX", "I\u{00AD}X", None, true).await;
     auth_test_options("\u{2168}", "IV", None, true).await;
     auth_test_options("\u{2168}", "I\u{00AD}V", None, true).await;
-}
-
-#[cfg_attr(feature = "tokio-runtime", tokio::test)]
```

Comment:
I'm confused about why this was deleted; what does this test have to do with `run_command` errors?

Reply:
The setup created the same user twice, which was always failing. The assertions are preserved, just the duplicated setup was removed.

Reply:
Okay, I see. I was confused by the fact that the two tests were merged into one.